src/connector/signaling/signaling-client.ts

```diff
@@ -53,7 +54,10 @@ export const SignalingClient = (input: {
   const subjects = input.subjects
   const subscription = new Subscription()
   const connectionId = Buffer.from(input.secrets.connectionId).toString('hex')
-  const url = `${input.baseUrl}/${connectionId}?target=${input.target}&source=${input.source}`
+  const url = `${
+    RadixConnectConfigurations[input.radixConnectConfiguration]
```

Comment:
This does not allow us to set an arbitrary value for signaling server URL, e.g. testing dev environments. IMO it should be possible to set SS from environment variables